doc/api/deprecations.md

```diff
@@ -40,7 +40,7 @@ However, the deprecation identifier will not be modified.
 
 ## List of deprecated APIs
 
-<a id="DEP0001"></a>
+<i id="DEP0001"></i>
```

Comment:
```suggestion
<span id="DEP0001"></span>
```

Reply:
Hmm, actually, I think `<span>` might be worse from an accessibility perspective?